packages/gestalt/src/Modal.js

```diff
@@ -89,32 +105,30 @@ export default function Modal({
           className={styles.container}
           role={role}
         >
-          <Backdrop>
-            <OutsideEventBehavior onClick={handleOutsideClick}>
```

Comment:
main change is to no longer use `OutsideEventBehavior` here - we add the `onClick` on the `Backdrop` instead